src/helpers.js

```diff
@@ -141,6 +144,19 @@ const runLinter = (editor: TextEditor) =>
     ? atom.commands.dispatch(atom.views.getView(editor), LINTER_LINT_COMMAND)
     : undefined);
 
+const getDebugInfo = () => {
+  const prettierAtomInfo = readPkg.sync();
+  const prettierInfo = readPkg.sync(getDepPath('prettier'));
+  const prettierESLintInfo = readPkg.sync(getDepPath('prettier-eslint'));
```

Comment:
can we inline these in the object being returned, that way we can avoid having any local state?